benchmarks/dungeon-benchmark/build.gradle.kts

```diff
@@ -0,0 +1,52 @@
+plugins {
+  id("com.android.test")
+  id("org.jetbrains.kotlin.android")
+}
+
+// Note: We are not including our defaults from .buildscript as we not need the base Workflow
+// dependencies that those include.
+
```

Comment:
Are we also tied to these sdk versions?

Reply:
The ones specified here you mean? Or the ones in the base files in `.buildscript`? We are not tied to any for running the benchmarks per se.